composer.json

```diff
@@ -37,15 +37,16 @@
         "nette/robot-loader": "^3.4",
         "nette/utils": "^3.2",
         "nikic/php-parser": "^4.10.4",
-        "phpstan/phpstan": "^0.12.83",
         "phpstan/phpdoc-parser": "^0.5.4",
+        "phpstan/phpstan": "^0.12.83",
         "phpstan/phpstan-phpunit": "^0.12.18",
-        "rector/rector-symfony": "^0.10.3",
-        "rector/rector-nette": "^0.10.5",
-        "rector/rector-laravel": "^0.10.1",
-        "rector/rector-cakephp": "^0.10.3",
-        "rector/rector-phpunit": "^0.10.6",
+        "rector/rector-cakephp": "^0.10.4",
         "rector/rector-doctrine": "^0.10.2",
+        "rector/rector-installer": "dev-main",
```

Comment:
Could you tag this one too, so we can remove min-stability dev?

Reply:
I have also bumped it up to PHP Version 7.3